sample_files/pre-commit-13.0/.pre-commit-config.yaml

```
@@ -85,10 +85,6 @@ repos:
     rev: v1.26.2
     hooks:
       - id: pyupgrade
-  - repo: https://github.com/asottile/seed-isort-config
-    rev: v1.9.4
-    hooks:
-      - id: seed-isort-config
   - repo: https://github.com/pre-commit/mirrors-isort
     rev: v4.3.21
```

We don't need to use the mirror now:

```suggestion
  - repo: https://github.com/timothycrosley/isort
    rev: 4.3.21
```

---

4.3.21-2 actually

---

No, it has to be 4.3.21 to be avoid the workaround from https://github.com/OCA/maintainer-quality-tools/pull/629/files#diff-142d2ae5d311d03a7bda3ae61bd6572dR95 AFAICS.

---

Hm, the 4.3.21 tag in that repo is broken as I understand, since it includes some 5.0 stuff: https://github.com/timothycrosley/isort/blob/4.3.21/CHANGELOG.md. So let's stick to the mirrors repo for now (b6c3c2e5b11d7037f8dda37fdc40132e6d0972bf). We'll re-evaluate when isort 5 is out.